### PR TITLE
LINK-1454 | Select price group for a signup

### PR DESCRIPTION
**Pull request by jorilindell**
## Description
Select price group for a signup

## Closes
[LINK-1454](https://helsinkisolutionoffice.atlassian.net/browse/LINK-1454)

[LINK-1454]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-1454?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: public/locales/fi/signup.json [hunk 1]
```diff
@@ -99,6 +99,7 @@
     "labelFirstName": "Etunimi",
     "labelLastName": "Sukunimi",
     "labelPhoneNumber": "Puhelinnumero",
+    "labelPriceGroup": "Hintaryhmä",
```
Comment (harriris-vincit) [inline public/locales/fi/signup.json:102]: The Figma prototype uses the term "Asiakasryhmä" - was that maybe agreed to be changed to "Hintaryhmä" or is this something to change?

File: public/locales/fi/signup.json [hunk 2]
```diff
@@ -107,6 +108,7 @@
     "placeholderFirstName": "Syötä ilmoittautujan etunimi",
     "placeholderLastName": "Syötä ilmoittautujan sukunimi",
     "placeholderPhoneNumber": "+358 44 1234 567",
+    "placeholderPriceGroup": "Valitse hintaryhmä",
```
Comment (harriris-vincit) [inline public/locales/fi/signup.json:111]: Here possibly "asiakasryhmä" should be used

File: public/locales/fi/signup.json (hunk 1 above)
Comment (jorilindell) [reply public/locales/fi/signup.json]: Changed "Hintaryhmä" to "Asiakasryhmä"

File: public/locales/fi/signup.json (hunk 2 above)
Comment (jorilindell) [reply public/locales/fi/signup.json]: Changed "Hintaryhmä" to "Asiakasryhmä"

File: public/locales/sv/signup.json [hunk 3]
```diff
@@ -99,6 +99,7 @@
     "labelFirstName": "Förnamn",
     "labelLastName": "Efternamn",
     "labelPhoneNumber": "Telefonnummer",
+    "labelPriceGroup": "Prisgrupp",
```
Comment (harriris-vincit) [inline public/locales/sv/signup.json:102]: Now that the Finnish term is "Asiakasryhmä", the Swedish and English terms should probably be in line with that ("Kundgrupp" for Swedish and "Customer Group" for English).

I need to do the same change for the back-end as I've used "price group" there too.

**Review by harriris-vincit — APPROVED** [review]
Left a comment about the English and Swedish terms for price group / "Asiakasryhmä", but otherwise look good.

**Pull request merged**